crates/optimism/bin/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "op-reth"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[dependencies]
+reth-node-builder.workspace = true
+reth-cli-util.workspace = true
+reth-optimism-cli.workspace = true
+reth-tracing.workspace = true
+reth-node-optimism.workspace = true
```

Comment:
in response to https://github.com/paradigmxyz/reth/pull/9439#issuecomment-2227409172:

you will have to add an optimism feature to this manifest, and then look through these deps, and see which has an optimism feature...so they can be included, just like for the other optimism crates - eventually we hope to get rid off this!